codespell_lib/data/dictionary.txt

```diff
@@ -23291,7 +23298,7 @@ throuh->through
 throuth->through
 throwed->threw, thrown,
 throwgh->through
-thru->through, thru,
+thru->through, thru, disabled because one may want to allow informal spelling
```

Comment:
I think we should drop the replacement of itself, in fact I thought I added some code so Travis would catch such anomalies, but perhaps it ignores the disabled ones.

Reply:
Done. There are definitely currently a bunch of current cases where the original spelling is marked as one of the possible good spellings of a potential misspellings.

Reply:
Yeah agreed. What's your take on this @larsoner ? I've just added #1437 to continue to try and clean up the dictionary.

The first one it's found is:
`arithmetics->arithmetic, arithmetics,`

Should this be banned throughout, or allowed where it's disabled (for suggestions) or what?

This particular example I'd agree is probably fairly rare in plural form, but there's no note to advise the person doing a manual correction of that, and why its flagged in a non-interactive session. Another multi-dictionary candidate?
https://www.merriam-webster.com/thesaurus/arithmetics

Reply:
@qyearsley . Sorry to keep dragging you into this, if you revert this change, I'm personally happy to merge as is. Otherwise I'd rather get some general consistency. Also please do feel free to re-open a new PR with the names stuff/or raise an issue, so some consensus can be reached.

Reply:
No problem. I've now reverted the change on this line.

Reply:
This is now in https://github.com/codespell-project/codespell/blob/master/codespell_lib/data/dictionary_informal.txt